architecture/libraries.rst

```diff
@@ -4,6 +4,21 @@
 Libraries
 #########
 
+.. _libutils:
+
+libutils
+*******
+
+OP-TEE core and OP-TEE dev-kit for Trusted Applciation development does not
```

Comment:
I'd rather say that libutils _is_ the libc of OP-TEE core and dev-kit. The only fundamental difference is the name.

Reply:
'libc' ! that was the wording I was looking for.
I'll reword and straight force push an updated commit if you don't mind.